tools/cldr-code/src/main/java/org/unicode/cldr/test/CheckNumbers.java

```diff
@@ -237,34 +237,15 @@ public CheckCLDR handleCheck(
             }
         }
 
-        // Check that symbols are for an explicit number system;
```

Comment:
Why is this being dropped? We still want to ensure that .../symbols has a numberSystem.

Reply:
My mistake! I confused this section with the one below that's deleted. I'll fix that now.